lib/clients/BaseClient.py

```diff
@@ -531,6 +531,48 @@ def decrypt_and_extract_tarball_of_directory(self, encrypted_tarball_name, direc
             return result
         return None
 
+    def create_tarball_of_directory(self, directory_to_tar, tarball_name):
+        """Create a tarball of a directory.
+
+        :param directory_to_tar: the path to the directory to be archived
+        :param tarball_name: the path where to store the resulting archive
+
+        :Example:
+            ::
+                tar cvzpf ACS-Demo.tar.gz -C ACS-Demo .
```

Comment:
What this line is added for? This is what function is supposed to do right and not done by caller?